### ISSUE TITLE
Firmware update / 1.12 | 1.41

### ISSUE THREAD

**Issue by KinoSeed** [ottokiksmaler/nx500_nx1_modding#39]
bitrate addresses look like are being shifted by 0x4f000, however it looks like there is different type of calculations now, and to top it off, it looks like even if you change those, they don't matter anymore (and are loaded in a different way)

changing resolution still works (on new addresses): 

VGA to 2.5K
./poker $(pgrep di-camera-app) 0xe441c:5a8ea0e3 0xe4420:0aaca0e3

DC to 2.5K
./poker $(pgrep di-camera-app) 0xe43ec:5a8ea0e3 0xe43f0:0aaca0e3

basically everything but the bitrate updates is good to go

**Comment by Tommixoft** [comment]
i wont risk updating FW - i see no reason for this update other than trying to harm current progress :)

**Comment by ottokiksmaler** [comment]
Different libraries in /usr/lib
```
325596 Mar 22 05:07 libsap-api.so.1.0.0
108208 Mar 22 05:07 libsap-client-stub-api.so.1.0.0
 18360 Mar 30 06:34 libversion-info.so
```
Old:
```
122660 May 22  2015 libsap-api.so.1.0.0
 49904 May 22  2015 libsap-client-stub-api.so.1.0.0
 18360 Aug 10  2015 libversion-info.so
```
/usr/apps/com.samsung.di-camera-app/bin
```
4773516 Mar 29 08:31 di-camera-app
```
Old:
```
4771528 Jul 30  2015 di-camera-app
```
/usr/bin
```
106656 Mar 22 05:07 sap-server-1.0.0
```
Old:
```
35084 May 22  2015 sap-server-1.0.0
```

**Comment by ottokiksmaler** [comment]
diff list mnt-v1.11 mnt2-v1.12:
```
Files mnt/etc/device-sec-policy and mnt2/etc/device-sec-policy differ
Files mnt/etc/info.ini and mnt2/etc/info.ini differ
Files mnt/etc/machine-id and mnt2/etc/machine-id differ
Files mnt/etc/opt/upgrade/500.sap.db.patch.sh and mnt2/etc/opt/upgrade/500.sap.db.patch.sh differ
Files mnt/etc/passwd- and mnt2/etc/passwd- differ
Only in mnt2/etc/systemd/default-extra-dependencies/ignore-units.d: ft.service
Only in mnt2/etc/systemd/default-extra-dependencies/ignore-units.d: ignore
Only in mnt2/etc/systemd/default-extra-dependencies/ignore-units.d: sap.service
Files mnt/etc/version.info and mnt2/etc/version.info differ
Files mnt/usr/apps/com.samsung.di-camera-app/author-signature.xml and mnt2/usr/apps/com.samsung.di-camera-app/author-signature.xml differ
Files mnt/usr/apps/com.samsung.di-camera-app/bin/di-camera-app and mnt2/usr/apps/com.samsung.di-camera-app/bin/di-camera-app differ
Files mnt/usr/apps/com.samsung.di-camera-app/signature1.xml and mnt2/usr/apps/com.samsung.di-camera-app/signature1.xml differ
Files mnt/usr/apps/org.tizen.browser/data/db/.browser.db-journal and mnt2/usr/apps/org.tizen.browser/data/db/.browser.db-journal differ
Files mnt/usr/apps/org.tizen.browser/data/db/.html5-custom-handler.db-journal and mnt2/usr/apps/org.tizen.browser/data/db/.html5-custom-handler.db-journal differ
Files mnt/usr/bin/sap-server and mnt2/usr/bin/sap-server differ
Files mnt/usr/bin/sap-server-1.0.0 and mnt2/usr/bin/sap-server-1.0.0 differ
Files mnt/usr/lib/libfilelist-manager.so and mnt2/usr/lib/libfilelist-manager.so differ
Files mnt/usr/lib/libsap-api.so and mnt2/usr/lib/libsap-api.so differ
Files mnt/usr/lib/libsap-api.so.1 and mnt2/usr/lib/libsap-api.so.1 differ
Files mnt/usr/lib/libsap-api.so.1.0.0 and mnt2/usr/lib/libsap-api.so.1.0.0 differ
Files mnt/usr/lib/libsap-client-stub-api.so and mnt2/usr/lib/libsap-client-stub-api.so differ
Files mnt/usr/lib/libsap-client-stub-api.so.1 and mnt2/usr/lib/libsap-client-stub-api.so.1 differ
Files mnt/usr/lib/libsap-client-stub-api.so.1.0.0 and mnt2/usr/lib/libsap-client-stub-api.so.1.0.0 differ
Files mnt/usr/lib/pango/1.8.0/module-files.d/pango.modules and mnt2/usr/lib/pango/1.8.0/module-files.d/pango.modules differ
Files mnt/usr/lib/systemd/system/sap.service and mnt2/usr/lib/systemd/system/sap.service differ
Only in mnt/usr/lib/xtables: ac-plugins
Only in mnt/usr/lib/xtables: alsa-lib
Only in mnt/usr/lib/xtables: bank
Only in mnt/usr/lib/xtables: bluetooth
Only in mnt/usr/lib/xtables: bufmgr
Only in mnt/usr/lib/xtables: dbus-1
Only in mnt/usr/lib/xtables: debug
Only in mnt/usr/lib/xtables: driver
Only in mnt/usr/lib/xtables: ecore
Only in mnt/usr/lib/xtables: edje
Only in mnt/usr/lib/xtables: efreet
Only in mnt/usr/lib/xtables: elementary
Only in mnt/usr/lib/xtables: elfutils
Only in mnt/usr/lib/xtables: engines
Only in mnt/usr/lib/xtables: enlightenment
Only in mnt/usr/lib/xtables: evas
Only in mnt/usr/lib/xtables: firmware
Only in mnt/usr/lib/xtables: games
Only in mnt/usr/lib/xtables: gconv
Only in mnt/usr/lib/xtables: gio
Only in mnt/usr/lib/xtables: gstreamer-0.10
Only in mnt/usr/lib/xtables: libaccounts-svc.so.0
Only in mnt/usr/lib/xtables: libaccounts-svc.so.0.2.78
Only in mnt/usr/lib/xtables: libace-client.so
Only in mnt/usr/lib/xtables: libace-client.so.0
Only in mnt/usr/lib/xtables: libace-client.so.0.0.67
Only in mnt/usr/lib/xtables: libace-dao-ro.so
Only in mnt/usr/lib/xtables: libace-dao-ro.so.0
Only in mnt/usr/lib/xtables: libace-dao-ro.so.0.0.67
Only in mnt/usr/lib/xtables: libace-dao-rw.so
Only in mnt/usr/lib/xtables: libace-dao-rw.so.0
Only in mnt/usr/lib/xtables: libace-dao-rw.so.0.0.67
Only in mnt/usr/lib/xtables: libace-install.so
Only in mnt/usr/lib/xtables: libace-install.so.0
Only in mnt/usr/lib/xtables: libace-install.so.0.0.67
Only in mnt/usr/lib/xtables: libace-popup-validation.so
Only in mnt/usr/lib/xtables: libace-popup-validation.so.0
Only in mnt/usr/lib/xtables: libace-popup-validation.so.0.0.67
Only in mnt/usr/lib/xtables: libace-settings.so
Only in mnt/usr/lib/xtables: libace-settings.so.0
Only in mnt/usr/lib/xtables: libace-settings.so.0.0.67
Only in mnt/usr/lib/xtables: libace.so
Only in mnt/usr/lib/xtables: libace.so.0
Only in mnt/usr/lib/xtables: libace.so.0.0.67
Only in mnt/usr/lib/xtables: libail.so.0
Only in mnt/usr/lib/xtables: libail.so.0.1.0
Only in mnt/usr/lib/xtables: libalarm.so.0
Only in mnt/usr/lib/xtables: libalarm.so.0.0.0
Only in mnt/usr/lib/xtables: liballshare-dms.so
Only in mnt/usr/lib/xtables: liballshare-dms.so.0
Only in mnt/usr/lib/xtables: liballshare-dms.so.0.1
Only in mnt/usr/lib/xtables: liballshare.so
Only in mnt/usr/lib/xtables: liballshare.so.1
Only in mnt/usr/lib/xtables: liballshare.so.1.4.0
Only in mnt/usr/lib/xtables: libanl_pic.map
Only in mnt/usr/lib/xtables: libapex.so
Only in mnt/usr/lib/xtables: libapp2ext.so.0
Files mnt/usr/sbin/wpa_supp.sh and mnt2/usr/sbin/wpa_supp.sh differ
Files mnt/usr/share/usr_share_locale.squash and mnt2/usr/share/usr_share_locale.squash differ
Files mnt/var/cache/ldconfig/aux-cache and mnt2/var/cache/ldconfig/aux-cache differ
Files mnt/var/cache/zypp/solv/@System/cookie and mnt2/var/cache/zypp/solv/@System/cookie differ
Files mnt/var/lib/systemd/random-seed and mnt2/var/lib/systemd/random-seed differ
Files mnt/var/lib/zypp/AnonymousUniqueId and mnt2/var/lib/zypp/AnonymousUniqueId differ
Only in mnt2/var/run/memory/private/sap: rfcomm_ready
```

**Comment by vasile-gh** [comment]
@ottokiksmaler - if the above are for me, no need - I will probably not need them for the bitrate... rgds

**Comment by ottokiksmaler** [comment]
Heh, there's no mention of nx_dev.adj in new di-camera-app file :(

**Comment by vasile-gh** [comment]
@ottokiksmaler not too worried about that:
```(zeus:vasile)-(jobs:0)-(~/nx/nx500)
(! 510)-> sudo grep -r -i nx_dev.adj mnt-v1.1*
Binary file mnt-v1.11/rootfs/usr/apps/com.samsung.di-camera-app/bin/di-camera-app matches
Binary file mnt-v1.11/rootfs/usr/lib/libproduction-mode.so matches
Binary file mnt-v1.11/part/nx500-mmcblk0p9----snapshot.bin matches
Binary file mnt-v1.11/part/nx500-mmcblk0p10---platform.bin matches
Binary file mnt-v1.12/rootfs/usr/lib/libproduction-mode.so matches
Binary file mnt-v1.12/part/nx500-mmcblk0p10---platform.bin matches

**Comment by vasile-gh** [comment]
Looks like they **did not** limit themselves to doing Android 6.0-related stuff.
Good news :-) [for the future of NX line, that it]. Bad news for **me**, right now.

**Comment by ottokiksmaler** [comment]
It's not in snapshot either. Could you check if it works in 1.12?

**Comment by vasile-gh** [comment]
does not work in 1.12 but I will work around this.
